var/spack/repos/builtin/packages/intel-mpi-benchmarks/package.py

```diff
@@ -18,31 +18,52 @@ class IntelMpiBenchmarks(MakefilePackage):
     """
 
     homepage = "https://software.intel.com/en-us/articles/intel-mpi-benchmarks"
-    url = "https://github.com/intel/mpi-benchmarks/archive/IMB-v2019.5.tar.gz"
```

Comment:
Can you keep the `url` here? `url_for_version` is used for downloads, but `url` is used to search for new versions. So things like `spack versions` and `spack checksum` only work if there is some `url` available. You can just use the newer URL format, people usually only care about adding new releases.